.github/workflows/mindsdb.yml

```diff
@@ -172,10 +204,6 @@ jobs:
         SOURCE_DIR: 'distributions/linux/dist'
         DEST_DIR: 'mindsdb-installer/linux'
 
-    - name: Build OSX sh installer
-      run: |
-        python distributions/osx/rename_installers.py beta
```

Comment:
Don't remove this, we can still keep the script installer.

Reply:
Done